logbook-core/src/main/java/org/zalando/logbook/BaseHttpMessage.java

```diff
@@ -20,8 +20,6 @@
  * #L%
  */
 
-import com.google.common.collect.Multimap;
-
 import java.nio.charset.Charset;
 
 public interface BaseHttpMessage {
```

Comment:
Another idea, maybe it would be better to introduce a `Headers` class to encapsulate this behaviour. This might also make it easier to make sure all implementations handle headers case-insensitive as in PR #73. It might even share some code with `QueryParameters` regarding obfuscation.